plugins/reporters/web-app-template/src/components/VulnerabilityRatingTag.jsx

```diff
@@ -0,0 +1,41 @@
+/*
+ * Copyright (C) 2025 The ORT Project Authors (see <https://github.com/oss-review-toolkit/ort/blob/main/NOTICE>)
```

Comment:
BTW, here you're using 2025, but in other added files 2024. Should we align on 2025 even if some of the commit are older?